packages/components/carousels/SmallCarousel/SmallCarouselAlt.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC } from "react";
+import { View, ViewStyle } from "react-native";
+import Carousel, { TCarouselProps } from "react-native-reanimated-carousel";
+
+import { SVG } from "../../SVG";
+
+import chevronLeftSVG from "@/assets/icons/chevron-left.svg";
+import chevronRightSVG from "@/assets/icons/chevron-right.svg";
+import { CustomPressable } from "@/components/buttons/CustomPressable";
+import { useSmallCarousel } from "@/components/carousels/SmallCarousel/useSmallCarousel";
+import { SpacerRow } from "@/components/spacer";
+import { neutral33 } from "@/utils/style/colors";
+import { layout } from "@/utils/style/layout";
+
+// SmallCarousel but with rounded buttons instead of shadowed sides buttons
```

Comment:
Could we merge this with SmallCarousel ? I see there is quite a few shared codes

Reply:
I already merged stuff into this hook: https://github.com/TERITORI/teritori-dapp/blob/feat-map-posts/packages/components/carousels/SmallCarousel/useSmallCarousel.ts#L11
I don't see what I can merge from `SmallCarousel` and `SmalCarouselAlt` without adding more complexity